MediaBrowser.Server.Mono/ApplicationPathHelper.cs

```diff
@@ -1,44 +1,56 @@
 ﻿using System;
-using System.Configuration;
 using System.IO;
 using System.Runtime.InteropServices;
+using EmbyServer;
 
 namespace MediaBrowser.Server.Mono
 {
-    public static class ApplicationPathHelper
+    public class ApplicationPathHelper
```

Comment:
Moved from a static class to an instanced class, as it's very hard to test static states. Alternatively we could leave this as static and pass more information into the `GetProgramData(...)` method through its parameters.